### PR TITLE
Allow for device service and port to be configured

### PR DESCRIPTION
**Pull request by petertuton**
## Purpose
Purpose is to allow the service and port on the device to be configurable (default is http).

## Does this introduce a breaking change?
```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Deploy resources
As per the README. In settings.yaml, change the values for `serviceProtocol` and `servicePort`, e.g., 
```
serviceProtocol: ssh
servicePort: 22
```

* Deploy the resources
As per README instructions...

* Test the code
Once all resources are deployed, check the `app.config` file to ensure the addition keys are added and match the expected values, e.g.:
```
<add key="SERVICE_PROTOCOL" value="ssh" />
<add key="SERVICE_PORT" value="22" />
```
Remaining steps as per the README instructions...

Check the response from the POST request to the Azure Function includes the service protocol value. 

Check the device service can be accessed via the appropriate client device, e.g.:
```
ssh -p 8090 <username>@aci.fqdn
```

## Other Information
If the connection times-out, wait some time to ensure the ACI deployment is complete. Also ensure the device's service is accessible locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +52,9 @@ The flow below demonstrates how a user can access a web server that is running o
     cp sample.settings.yaml settings.yaml
     ```
 
-    set the tenantId and subscriptionId, and adjust the values for the Azure resource names by replacing `[unique-id]` with a unique identifier so that your resource names are unique across Azure.
+    Set the tenantId and subscriptionId, and adjust the values for the Azure resource names by replacing `[unique-id]` with a unique identifier so that your resource names are unique across Azure.
+
+    To use a different protocol than HTTP, change the value of `deviceProtocol` to an appropriate value, e.g., `ssh` and change the `devicePort` to the port on which the service is running, e.g., `22`. Note: The value of the protocol is arbitrary, used only for messaging purposes and does not have to match the actual protocol running on the device.
```
Comment (norelina) [inline README.md]:
```suggestion
    To use a different protocol than HTTP, change the value of `serviceProtocol` to an appropriate value, e.g., `ssh` and change the `servicePort` to the port on which the service is running, e.g., `22`. Note: The value of the protocol is arbitrary, used only for messaging purposes and does not have to match the actual protocol running on the device.
```

**Review by norelina — COMMENTED** [review]
Thank you @petertuton for this amazing contribution!

File: README.md [hunk 2]
```diff
@@ -97,10 +101,12 @@ To start the simulated device follow these steps:
 
     ![simulated-device-telemetry](docs/assets/simulated-device-telemetry.png)
 
-1. Confirm you can access the local web server on `http://localhost:8080/`
+1. For http, confirm you can access the local web server on `http://localhost:8080/`
 
     ![web-server](docs/assets/web-server.png)
 
+    If you are using a different protocol, use the appropriate client tool to access the service running on the device.
```
Comment (norelina) [inline README.md]: Can we also include an example on how to verify when using ssh?

Comment (petertuton) [reply README.md]: Done!

**Review by norelina — APPROVED** [review]

**Pull request merged**